samples/client/petstore/swift5/resultLibrary/PetstoreClient/Classes/OpenAPIs/Models/EnumArrays.swift

```diff
@@ -13,7 +13,7 @@ import AnyCodable
 public struct EnumArrays: Codable, Hashable {
 
     public enum JustSymbol: String, Codable, CaseIterable {
-        case greaterThanOrEqualTo = ">="
+        case greaterThanEqual = ">="
```

Comment:
@mbruegmann I think those changes are not expected, or are they?
Could you please check what happened?

Reply:
I hav no idea what this is for. Those were the only changes I got after executing the given 3 commands.